chia/server/server.py

```diff
@@ -278,6 +278,11 @@ async def incoming_connection(self, request):
                 error_stack = traceback.format_exc()
                 self.log.error(f"Exception {e}, exception Stack: {error_stack}")
                 close_event.set()
+        except ValueError as e:
```

Comment:
i think we can unify this with the ProtocolError case "except (ProtocolError, ValueError), e:"

Reply:
Wouldn't this require something like if (isinstance(e, ProtocolError)) - which doesn't seem any cleaner?